data/transactions/logic/eval.go

```diff
@@ -91,6 +91,29 @@ func stackValueFromTealValue(tv *basics.TealValue) (sv stackValue, err error) {
 	return
 }
 
+// ComputeMinTealVersion calculates the minimum safe TEAL version that may be
+// used by a transaction in this group. It is important to prevent
+// newly-introduced transaction fields from breaking assumptions made by older
+// versions of TEAL. If one of the transactions in a group will execute a TEAL
+// program whose version predates a given field, that field must not be set
+// anywhere in the transaction group, or the group will be rejected.
+func ComputeMinTealVersion(group []transactions.SignedTxn) uint64 {
+	var minVersion uint64
```

Comment:
init to 1 ? I like version 1 better then 0

Reply:
I don't think we can do that, because then we'll always reject v0 programs.

Reply:
^+1